releasenotes/notes/ignore_under_parameter-e06eab711a713571.yaml

```diff
@@ -0,0 +1,14 @@
+---
+features_visualization:
+  - |
+    New parameter added to the function plot_histogram. Parameter is named `ignore_under`. 
+    The introduction of this parameter ignores the values in the data dictionary values
+    strictly lower than `ignore_under`. This will solve the sparsity problem faced by the
+    function when the data contains a lot of small values returning an unreadable histogram.
+
+
+fixes:
```

Comment:
This is a new feature, it's not changing existing behavior, so it's not really a bug fix. The above note should be enough 👍🏻